AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/Dependency/AWSPinpointAdapter+AnalyticsClientBehavior.swift

```diff
@@ -7,6 +7,7 @@
 
 import AWSPinpoint
 import Foundation
+import StoreKit
```

Comment:
curious why StoreKit is needed here?

Reply:
For `SKPaymentTransaction` and `SKProduct`